graphql-samples/mutations/collaboration/create-aspect-on-callout

```diff
@@ -0,0 +1,19 @@
+mutation createAspectOnCallout($data: CreateAspectOnCalloutInput!){
```

Comment:
This mutations doesn't have the correct authorization set. Global admin should have sufficient privilleges to create aspect on 3rd party callout (one that wasn't created by the global admin). That's not the case (tested after successful auth policy reset).

![image](https://user-images.githubusercontent.com/7090348/183408910-0cfa0045-29b2-4aaf-840a-270c9eaa27d7.png)